doc/source/contributing_code.txt

```diff
@@ -131,11 +131,14 @@ all issues have been resolved.
 <li>Programmers begin work on a feature or fix by branching from
 develop.
 
-<li>When branch is ready, it is submitted to code review.
+<li>When a branch is ready, it is submitted to code review.
```

Comment:
A single "integrator" is assigned to handle merging the PR.  Any developer my comment on a PR but only the integrator may decide to merge or not.  Developers should not integrate their own PRs.